workers/social/lib/social/models/account.coffee

```diff
@@ -301,8 +301,9 @@ module.exports = class JAccount extends jraphical.Module
 
 
   fetchRelativeGroups$: secure (client, callback) ->
-
-    @fetchRelativeGroups callback
+    delegate = client?.connection?.delegate
+    return callback new Error 'malformed request'
```

Comment:
actually this must be converted to `permit` usage or at least there must be another check for if requester is this account. this won't work like this. like; `if @equals delegate ...`